src/main/g8/application/src/main/scala/$package__packaged$/api/commands/DatabaseOptions.scala

```diff
@@ -1,15 +1,14 @@
 package $package$.api.commands
 
+import cats.effect._
 import cats.implicits._
+import com.lightbend.emoji.ShortCodes.Defaults._
+import com.lightbend.emoji.ShortCodes.Implicits._
 import com.monovore.decline.Opts
-import cats.effect._
-import doobie.util.transactor.Transactor
+import com.monovore.decline.refined._
 import doobie.implicits._
-import com.lightbend.emoji.ShortCodes.Implicits._
-import com.lightbend.emoji.ShortCodes.Defaults._
-import com.monovore.decline._
```

Comment:
Calling some extra attention to this one. This got flagged as an unused import after merging it with `com.monovore.decline.Opts`, so I removed it. The monovore docs often import the whole namespace, but I'm in favor of only grabbing what we need. We tend to chain off `Opts` in our usage so that should be sufficient for most cases.